internal/pkg/print/markdown/content/content.go

```diff
@@ -0,0 +1,144 @@
+package content
+
+import (
+	"bytes"
+	"fmt"
+	"strings"
+
+	"github.com/segmentio/terraform-docs/internal/pkg/doc"
+	"github.com/segmentio/terraform-docs/internal/pkg/print"
+	"github.com/segmentio/terraform-docs/internal/pkg/settings"
+)
+
+// Print prints a document as markdown content.
+func Print(document *doc.Doc, settings settings.Settings) (string, error) {
+	var buffer bytes.Buffer
+
+	if document.HasComment() {
+		printComment(&buffer, document.Comment, settings)
+	}
+
+	if document.HasInputs() {
+		if settings.Has(print.WithSortByName) {
+			if settings.Has(print.WithSortInputsByRequired) {
+				doc.SortInputsByRequired(document.Inputs)
+			} else {
+				doc.SortInputsByName(document.Inputs)
+			}
+		}
+
+		printInputs(&buffer, document.Inputs, settings)
+	}
+
+	if document.HasOutputs() {
+		if settings.Has(print.WithSortByName) {
+			doc.SortOutputsByName(document.Outputs)
+		}
+
+		if document.HasInputs() {
+			buffer.WriteString("\n")
+		}
+
+		printOutputs(&buffer, document.Outputs, settings)
+	}
+
+	return buffer.String(), nil
+}
+
+func printComment(buffer *bytes.Buffer, comment string, settings settings.Settings) {
```

Comment:
Please retain the order alphabetical order of functions.

Reply:
+1